docs/tutorials/serving/rest_simple.ipynb

```diff
@@ -355,7 +355,12 @@
       },
       "outputs": [],
       "source": [
-        "!{SUDO_IF_NEEDED} apt-get install tensorflow-model-server"
+        "#!{SUDO_IF_NEEDED} apt-get install tensorflow-model-server #installing Tensorflow Model server 2.8 instead of latest version\n"
```

Comment:
This seems like not a valid notebook format. ',' should be added at the end of the line...